rs/canonical_state/src/encoding/tests/compatibility.rs

```diff
@@ -1514,10 +1358,6 @@ fn request_metadata() -> RequestMetadata {
     RequestMetadata::new(13, Time::from_nanos_since_unix_epoch(101))
 }
 
-fn deadline(certification_version: CertificationVersion) -> CoarseTime {
-    if certification_version >= CertificationVersion::V18 {
-        CoarseTime::from_secs_since_unix_epoch(8)
-    } else {
-        NO_DEADLINE
-    }
+fn deadline(_certification_version: CertificationVersion) -> CoarseTime {
+    CoarseTime::from_secs_since_unix_epoch(8)
 }
```

Comment:
Without the diverging behavior depending on certification version, it may be cleaner to just inline the `CoarseTime::from_secs_since_unix_epoch(8)` wherever we currently call `deadline()`. As is, we're just obfuscating.